godotproject/EOL.gd

```diff
@@ -0,0 +1,9 @@
+extends Area2D
+
+func _on_EOL_body_entered(body):
+	# Change the current scene to the next level
+	if "is_god" in body: 
```

Comment:
Is this doing a check for the player? I think we should have a more consistent way of checking this

Reply:
The ant script uses this: `if RC.name == "Player":`
Not sure if we want to use that here as well, like:
```suggestion
	if body.name == "Player": 
```
I think that should probs work?

Reply:
Yeah I tested this locally and it worked 👍

Reply:
Good idea